crema/dataset.py

```diff
@@ -102,19 +109,43 @@ def __getitem__(self, column):
         """Return the specified column"""
         return self._data.loc[:, column]
 
-    def assign_confidence(
-        self, score_column=None, desc=None, eval_fdr=0.01, method="tdc",
-    ):
-        """Assign confidences estimates to this collection of PSMs.
+    def add_peptide_pairing(self, pairing):
+        """Adds a target/decoy peptide pairing to this collection of PSMs
 
+        Parameters
+        ----------
+        pairing : dict
+            A dictionary containing the target/decoy mapping to be used
+            during TDC
 
         """
+        if pairing is None:
+            return
+        if isinstance(pairing, dict):
```

Comment:
I removed it for now because I'm not sure what "other checks" I should be doing. I mainly did the type checking so users can't add a peptide pairing that isn't in the form of a dictionary.